### DISCUSSION TITLE
Registration page is unavailable

### DISCUSSION THREAD

**Discussion by KimSambaFX** [firefly-iii/firefly-iii#6732] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely me, not you](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I am deploying the application using docker-compose, with pretty much the same settings as described in the docs for docker-compose and nginx reverse proxy.

At the first deploy, the application works normally, going to the url brings me to the registration page, and i was able to register and login with no issues.

I have then stopped the docker and restarted it, and the application now routes me directly to login page. I am able to login without any issues, and i have access to all features, even the mails are functionning. But there is no way to register anymore.

When i manually route to /register, i am getting a page with the following:

```
Sorry, an error occurred.
Registration is currently not available.

If you do not know how to handle this error, please open an issue on [GitHub](https://github.com/firefly-iii/firefly-iii/issues) or [send me a message](mailto:james@firefly-iii.org).

Follow this link back to the index.
```

Looking at the console with inspects shows nothing, the console is empty and clear.

I am using postgresql but mariadb makes the same issue. I am getting the same issue whether i use the proxy url with https, or using directly the url with http.

### Expected behaviour

I should be able to access register page

### Debug information

Debug information generated at 2022-12-20 17:44:34 Canada/Eastern for Firefly III version **5.7.15**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.7.15 |
| Firefly III API | 1.5.6 |
| Build | 298, 02-11-2022 05:00:04 CET |
| DB version | 18 (exp. 18) |
| PHP | `8.1.11` |
| Host | `Linux` |

| System info | Value |
| --- | --- |
| System TZ | Canada/Eastern |
| Browser TZ | America/Toronto |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | info, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | apache2handler |
| Default language | en_US |
| Default locale | equal |
| BCscale | 24 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | pgsql |
| Trusted proxies (.env) | `**` |

| User info | Value |
| --- | --- |
| Login provider | eloquent  |
| User guard | web  |
| Headers | N/A, N/A  |
| Stateful domains |  |
| Session start | 2022-12-01 00:00:00 |
| Session end | 2022-12-31 23:59:59 |
| Session first | 2022-01-01 00:00:00 |
| User ID | 1 |
| User language | en_US |
| User locale | en_US |
| Attempt at "en_US.utf8" | 'en_US.utf8' |
| Attempt at "en_US.UTF-8" | 'en_US.UTF-8' |
| User agent | Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.127 Safari/537.36 |

### Steps to reproduce

1. delete any existing docker volume for a fresh install
2. run docker-compose up
3. go to the application and register
4. run docker-compose down
5. run docker-compose up
6. try to go to /register

### Additional info

The issue only happens if i create the first user.

**Comment by KimSambaFX** [comment]
I actually tested a bit more, and it seems it happens only after a first user is created

**Comment by JC5** [comment]
This is not a bug. Browse to the administration and enable registration again.

**Comment by JC5** [comment]
See also: https://docs.firefly-iii.org/firefly-iii/faq/general/#is-it-multi-user

**Comment by JC5** [comment]
My apologies, this "issue" is now a "discussion", I didn't mean to close it.

**Reply by KimSambaFX** [reply]
No problem and thank you very much for the quick answer.
It is indeed well documented in the FAQ.

However, i suggest it would be nice to display a link to that page in the "Registration disabled" page, i really thought it was an error given how the page was displayed and i tried to fix any issues i actually had with my reverse proxy when there was actually no issues at all.

**Reply by JC5** [reply]
I was thinking the exact same thing 😁
